pydarn/plotting/rtp.py

```diff
@@ -335,11 +340,21 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
                                      start_time=start_time,
                                      end_time=end_time,
                                      opt_beam_num=cls.dmap_data[0]['bmnum'])
-        if coord is Coord.SLANT_RANGE:
+                   
+        if coord is Coords.SLANT_RANGE:
             # Get rxrise from hardware files (consistent with RST)
             rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
                                     .hardware_info.rx_rise_time
             y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+        elif coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
+            rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
+                                    .hardware_info.rx_rise_time
+            y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+            Re = 6371
```

Comment:
Let me know to make a const file in another PR since I need this for my other PR and then we have consistency on constants.